Jellyfin.Server/Migrations/DisableTranscodingThrottling.cs

```diff
@@ -0,0 +1,32 @@
+using System;
```

Comment:
A separate folder/namespace for actual migration files vs. the migration infrastructure code might be nice

Reply:
Moved to `Migrations/Routines/`